### ISSUE TITLE
[Request] Allow player to put their ingots into the Economy

### ISSUE THREAD

**Issue by kreezxil** [WVAviator/IngotEconomy#1]
How about adding a feature whereby the player with Ingots of specified type (configurable) in their inventory can use a command that removes the ingots and adds that to their balance. Just the same they could pull the ingots out of the balance and back into their inventories. Everything should be configurable as to be disabled or having ratios set.

**Comment by WVAviator** [comment]
I was thinking about making a feature where players can use a command to "sell" specific items or "buy" those items as well. If I did it would be configurable by assigning values to item IDs in the config

**Comment by kreezxil** [comment]
Yes. That would fit very well this server side mod. And after that might you consider slowing server side player shops. Configurable of course to enable, disable, etc. ... Also admin shops. There is a Bukkit plugin that does this, I can find it if you need more information.

-----Original Message-----

**Comment by kreezxil** [comment]
That would be great 😊 

-----Original Message-----
